src/mappings/staking.ts

```diff
@@ -62,6 +69,61 @@ async function createWithdrawl({
   await withdrawl.save();
 }
 
+async function updateMaxUnstakeAmount(
```

Comment:
Move this to utils
[suggestion] change utils to a collection (a folder):
- pure helper functions like `cidToBytes32`, `decodeMetadata`
- help functions to update db, like `createIndexer`, `updateTotalStake`